rpc/client/client.go

```diff
@@ -123,7 +123,8 @@ func (c *Client) BroadcastTxCommit(ctx context.Context, tx types.Tx) (*ctypes.Re
 
 	// add to mempool and wait for CheckTx result
 	checkTxResCh := make(chan *abci.Response, 1)
-	err = c.node.Mempool.CheckTx(tx, func(res *abci.Response) {
+	mp := c.node.GetMempool()
+	err = mp.CheckTx(tx, func(res *abci.Response) {
```

Comment:
you can inline the code `c.node.GetMempool().CheckTx(` since there is only single use for `mp`
similarly for `c.node.GetP2P().GossipTx(`
for more than 1 use, you can have a separate variable defined